cmd/resource/list.go

```diff
@@ -47,18 +47,32 @@ func newListCommand() *cobra.Command {
 			limit := options.Global.Resource.List.Limit
 			skip := options.Global.Resource.List.Skip
 
-			user, err := cloud.Client().Me()
-			if err != nil {
-				output.Errorf(err.Error())
-			}
 			if kind == "cluster" {
+				user, err := cloud.Client().Me()
+				if err != nil {
+					output.Errorf(err.Error())
+				}
+
```

Comment:
This `cloud.Client().Me()` should not move to inside.

Reply:
Done